Tools/cases_generator/instructions.py

```diff
@@ -120,11 +120,13 @@ def __init__(self, inst: parsing.InstDef):
     def is_viable_uop(self) -> bool:
         """Whether this instruction is viable as a uop."""
         dprint: typing.Callable[..., None] = lambda *args, **kwargs: None
-        # if self.name.startswith("CALL"):
-        #     dprint = print
+        if "PY_EXACT" in self.name:
+            dprint = print
```

Comment:
Did you mean to leave this uncommented?

Reply:
Kind of, since it should actually be silent (`CALL_PY_EXACT_ARGS` is viable now).